webpack.config.js

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line import/extensions
+const config = require('10up-toolkit/config/webpack.config.js');
+
+const webpack = require('webpack');
+
+const WORDPRESS_NAMESPACE = '@wordpress/';
+const BUNDLED_PACKAGES = ['@wordpress/icons', '@wordpress/interface', '@wordpress/style-engine'];
```

Comment:
I'm not sure why this is needed. Are we essentially leaving these packages alone and not externalizing them? Meaning that we're bundling these three packages in our bundle?

Reply:
These Bundled packages are taken straight from the Dependency Extraction Webpack plugin. https://github.com/WordPress/gutenberg/blob/68d492a15be91cf621e91d5aa16d916ec7c1bd44/packages/dependency-extraction-webpack-plugin/lib/util.js#L2-L6

These packages are not included in WordPress core as dependencies and therefore need to be installed via npm if you want to use them. Therefore they should also not get externalized.